.github/workflows/ci.yml

```diff
@@ -12,3 +12,5 @@ jobs:
     uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v5
     with:
       mvnArgs: -U
+    secrets:
+      mvnArgs: -Dopenai.key=${{secrets.OPENAI_KEY}}
```

Comment:
Do we have data on the costs that we generate if we run this as proposed at least once a day?
I'd consider to run these tests with the real end2end endpoint only once a week or in month.